Yasgui/packages/yasr/src/plugins/table/extended-table.ts

```diff
@@ -174,7 +174,65 @@ export class ExtendedTable extends Table {
     }
   }
 
+  /**
+   * Checks if the specified <code>element</code> is truncated with ellipses.
+   *
+   * @param element The element to be checked for truncation.
+   * @return <code>true</code> if the element is truncated with ellipses, <code>false</code> otherwise.
+   */
+  private isTextEllipsized(element: HTMLElement) {
+    // If the scroll width is greater than the client width, content is likely truncated with an ellipsis
+    return element.scrollWidth > element.clientWidth;
+  }
+
+  private dataTableClickHandler(event) {
+    if (!this.persistentConfig.isEllipsed) {
+      // If the datatable is not in 'Compact view', then there is no need to take any action.
+      return;
+    }
+
+    let literalElement = event.target.closest('.literal-cell p');
+    if (literalElement) {
+      if (this.isTextEllipsized(literalElement)) {
```

Comment:
You can remove the nested if and just do `if (literalElement && isTextEllipsized) {...}`

Reply:
Done

Reply:
Change it in the tableMouseoverHandler as well!

Reply:
Done